AdobeStockClientApi/Api/ExceptionManagerInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockClientApi\Api;
+
+interface ExceptionManagerInterface
```

Comment:
Missing doc `api interfaces` must be documented it should be also an `@api` annotations there